design/ingressroute-design.md

```diff
@@ -74,6 +74,8 @@ spec:
       secretName: google-tls
       # optional: minimum TLS version this vhost should negotiate
       minimumProtocolVersion: "1.3"
+      # optional: enforces all requests are secured via https by returning an http 301 status code redirecting to https
+      enforceTLS: true
```

Comment:
nit: you're probably already doing this but this should probably be deleted so that there is no way of saying "don't TLS me bro", when spec.virtualhost.tls is present.

Reply:
yeah that's left over from the original design, thanks!